template/context.go

```diff
@@ -38,6 +38,10 @@ type Context struct {
 		Platform Platform
 	}
 
+	Engine struct {
```

Comment:
`Engine` is a property of the node, so it should be under node.

Reply:
Thank you for your feedback. `Engine` is moved under `Node`.